drivers/hmis/spec/requests/hmis/submit_form_spec.rb

```diff
@@ -11,6 +13,7 @@
 RSpec.describe Hmis::GraphqlController, type: :request do
   before(:all) do
     cleanup_test_environment
+    ::HmisUtil::JsonForms.seed_all
```

Comment:
need to reseed here, some previous test appears to delete the definitions. Not sure why